### ISSUE TITLE
pakreq: pptpd

### ISSUE THREAD

**Issue by LionNatsu** [AOSC-Dev/aosc-os-abbs#588]

**Comment by MingcongBai** [comment]
Please specify an upstream, there are multiple `pptpd` implementations.

**Comment by LionNatsu** [comment]
Poptop I think. That's the most frequent one when I bing'ed 'pptpd' and looked up at ArchLinux Wiki.

https://sourceforge.net/projects/poptop/

**Comment by MingcongBai** [comment]
This is a package I would be reluctant to package this, as the last version was released back in 2013, and there seemed not to be any form of active development. And also by their own description, PPTP is a "faulty protocol" - as mentioned here http://poptop.sourceforge.net/dox/protocol-security.phtml.

**Comment by LionNatsu** [comment]
Ye-----s, there's no doubt about the security of PPTP. It is not safe and faulty.Anyway, it is not a encrypting method, but a tunnel protocol. And it's maybe the most common protocol (infrastructure) used for tunneling​.*We should estimate security​ under certain attack model and security requirements.* In my case, I want to tunnel my network data from my mobile to my Raspberry Pi, then encrypt it and send it to The Outside, where my VPS is, decrypt it and act as an proxy.OpenVPN uses TUN/TAP virtual network interface to tunnel network into program. But OpenVPN's program is (always?) separated from operating systems. I have to get an App to control the tricky part of protocol. It's too heavy so far.BTW, I suggest use the master branch.

**Comment by MingcongBai** [comment]
Right... I will address this in the next update cycle. I have just wrapped up the current so... Enjoy.

**Comment by MingcongBai** [comment]
Almost ten years old and not addressed. Closing as stale.
